src/server/worker/master.go

```diff
@@ -409,11 +409,14 @@ func (a *APIServer) runJob(ctx context.Context, jobInfo *pps.JobInfo, pool *pool
 		}
 		tree := hashtree.NewHashTree()
 		var statsTree hashtree.OpenHashTree
+		var skippedTree hashtree.OpenHashTree
```

Comment:
I think I told you to implement it this way with an extra tree... but looking at it now I think you could just drop `skippedTree` and anywhere and use `statsTree` in its place. That way there's also no need to `Merge` since you're writing the files to the place they should ultimately go to.